packages/web-pkg/src/components/AppTemplates/AppWrapper.vue

```diff
@@ -186,6 +187,7 @@ export default defineComponent({
           driveAliasAndItem
         },
         query: {
+          ...unref(currentRoute).query,
           fileId: id,
           shareId: space.shareId,
           ...(unref(currentRoute).query?.app && { app: unref(currentRoute).query?.app }),
```

Comment:
This here and above becomes unnecessary with your change, I'm wondering if this is really what we want or if this might cause issues somewhere, e.g. with the context route query params...

Reply:
Good catch!

> I'm wondering if this is really what we want or if this might cause issues somewhere

You mean when re-setting all of the current route's query params? I decided against this initially. But apparently now other things break because params will get lost :/

Reply:
Yeah, maybe we can filter out some... iirc we had issues  in some bizzare place when we had implemented it like this inititially, maybe login/logout...

Reply:
Ah, misreading the diff.. issue happened in closeApp iirc. So if this is feally only overriding a few params but stays on the same page l otherwise, it might be fine.

Reply:
It seems so, at least I didn't find any other issues for now.